core_types/src/dna/zome.rs

```diff
@@ -79,6 +79,10 @@ pub struct Zome {
     #[serde(default)]
     pub capabilities: ZomeCapabilities,
 
+    /// An array of functions declared in this this zome.
+    #[serde(default)]
+    pub fn_declarations: ZomeFnDeclarations,
```

Comment:
Hm, I remember that the field `functions` in Capability once was called `fn_declarations` as well and we renamed it to `functions`.

I would actually prefer if this was `functions`, and functions in ZomeCapabilities would be renamed `function_names` as @maackle suggested above.

Reply:
```suggestion
    pub functions: ZomeFnDeclarations,
```